packages/libllvm/compiler-rt-ndk-version.diff

```diff
@@ -0,0 +1,11 @@
+--- ../projects-compiler-rt-CMakeLists.txt.orig	2020-09-22 14:12:54.771097201 +0000
++++ ./projects/compiler-rt/CMakeLists.txt	2020-09-22 16:11:07.568559345 +0000
+@@ -114,7 +114,7 @@
+ endif()
+ pythonize_bool(ANDROID)
+ 
+-set(ANDROID_NDK_VERSION 18
++set(ANDROID_NDK_VERSION @TERMUX_NDK_VERSION_NUM@
+     CACHE STRING "Set this to the Android NDK version that you are using")
```

Comment:
It would be much easier to just set `-DANDROID_NDK_VERSION=21` in `TERMUX_PKG_EXTRA_CONFIGURE_ARGS` above or does that mess with CMake in some other way?

Reply:
Yeah, that's easier. I was not sure if cmake passes configure args to the subprojects recursively but it seems to work indeed!